src/redemptions/redemption.service.spec.ts

```diff
@@ -139,133 +142,6 @@ describe('RedemptionServiceSpec', () => {
       );
     });
 
```

Comment:
Instead of deleting these tests, can you just update them to call `calculateStatus` instead of `isEligible` and update assertions?

Reply:
There are already tests for calculateStatus, these duplicated those tests.